test/cljs/rems/administration/test_create_form.cljs

```diff
@@ -310,6 +310,9 @@
                      [:form/fields 0 :field/placeholder :fi])
              :t.form.validation/required)))
 
+    (testing "placeholder is not validated if it is not used"
+      (is (empty? (validate-form (assoc-in form [:form/fields 0 :field/type] :label)))))
```

Comment:
Should this test set an invalid placeholder?

Reply:
yes...